src/main/java/sc/iview/SciView.java

```diff
@@ -247,17 +250,19 @@ protected void paintComponent(Graphics g) {
         try {
             Dimension screenSize = Toolkit.getDefaultToolkit().getScreenSize();
 
-            x = screenSize.width/2 - getWindowWidth()/2;
+            x = screenSize.width/2 - getWindowWidth()/2 - 300;
```

Comment:
This should probably go?

Reply:
Yes, you're right. Thanks for spotting it.